ui/src/cohortUpdaterContext.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { Cohort, GroupKind } from "cohort";
+import { ConceptCriteria } from "criteria/concept";
+import { CohortUpdater } from "./cohortUpdaterContext";
+
+test("updates", () => {
+  var cohort = new Cohort("name", "underlay", "entity");
```

Comment:
I don't think there are any cases where one should use `var` in ES6, as far as I'm aware. Would use `let` for any non-final variables